docs/Installing_IML_on_HPC_Storage_Sandbox.md

```diff
@@ -0,0 +1,143 @@
+[**Table of Contents**](index.md)
+
+# Installing IML on HPC Storage Sandbox
+
+![clustre](md_Graphics/installing_sm.jpg)
+
+## Prerequisites:
+Please refer to https://github.com/intel-hpdd/vagrantfiles on how to create a virtual HPC storage cluster with vagrant before attempting to install IML.
+
+## Notes:
+- You will be logging into your vagrant box as the vagrant user, which has the ability to run with root privilege. To elevate privileges to the root account, use the sudo command. The vagrant user does not require a password to run sudo. Should there ever be a need to login as root directly, the root password is also "vagrant". 
+
+    To acquire a root shell, use sudo in either one of the following invocations:
+    ```
+    sudo -s
+    ```
+    or
+    ```
+    sudo su -
+    ```
+
+## Installing IML:
+1. Verify the following vagrant plugins are installed:
+    ```
+    vagrant plugin install vagrant-shell-commander
+    ```
+2. Obtain the [preview 3 build](https://github.com/intel-hpdd/intel-manager-for-lustre/releases/tag/v4.0.0.0P3).
+3. Exit from the vagrant box and scp the build to the /tmp directory in your admin node. For example, if your admin node is running on port 2200 (you can verify this with `vagrant ssh-config`) and the build is in your Downloads folder:
+    ```
+    scp -P 2200 ~/Downloads/iml-4.0.0.0.tar.gz vagrant@127.0.0.1:/tmp/.
+    # password is "vagrant"
+    ```
+    Alternatively, use the output of `vagrant ssh-config` to create a temporary SSH configuration for the `scp` command. This is most useful when there will be a lot of interaction with a node from outside the VM:
+    ```
+    [user@mini iml]$ vagrant ssh-config | tee sshcfg| grep ^Host
+    Host adm
+    Host mds1
+    ...
+    [user@mini iml]$ scp -F sshcfg ~/Media/iml-4.0.0.0.tar.gz adm:/tmp/.
```

Comment:
Should reconcile `~/Media` here with `~/Downloads` up on line 30, just to keep your either/or examples consistent.

Reply:
Changed